models.py

```diff
@@ -146,13 +163,19 @@ def delete_custom_model(self):
         return self.STT.delete_language_model(self.get_customization_id())
 
     def train_custom_model(self):
-        return self.STT.train_language_model(self.get_customization_id())
+        resp = self.STT.train_language_model(self.get_customization_id())
+        self.wait_until('available', "Training")
+        print("Training complete. Custom model is ready to use.")
+        return resp
 
     def reset_custom_model(self):
         return self.STT.reset_language_model(self.get_customization_id())
 
     def upgrade_custom_model(self):
-        return self.STT.upgrade_language_model(self.get_customization_id())
+        resp = self.STT.upgrade_language_model(self.get_customization_id())
+        self.wait_until('available', "Upgrading")
+        print("Upgrade complete.")
```

Comment:
eprint please (uses stderr), allows redirect of stdout to file

Reply:
change made